templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml

```diff
@@ -302,6 +302,7 @@ contents:
 
     start_static_pods() {
       echo "Starting static pods.."
+      find ${MANIFEST_STOPPED_DIR} -maxdepth 1 -type f -printf "%f\n" > $STOPPED_STATIC_PODS
       while read STATIC_POD; do
```

Comment:
My fear of doing this is that we are possibly starting pods which we did not stop. Meaning after this dir $MANIFEST_STOPPED_DIR is created it is possible that additional static pods get placed in this dir. For example the node gets restored then in 6 months restored again. It makes me nervous that we might start something explicitly stopped by admin by placing in this dir.

I think if we should make stop_static_pods smarter. Check if $STOPPED_STATIC_PODS exists and if so don't overwrite/clobber. Then that list should be true and we can reset to that state. What do you think?

Reply:
I don't think we expect anyone else to reuse `/etc/kubernetes/manifests-stopped`, this is not a standard dir for stopped pods (the point is to move away from `/etc/kubernetes/manifests`).

A safer alternative would be `mktemp` I guess

Reply:
> I don't think we expect anyone else to reuse /etc/kubernetes/manifests-stopped

Right unexpected usage is what I was worried about. Another idea is backup the manifests and then also stop them locally in assets/$STOPPED_DIR. This way the stop dir is never created in /etc/kubernetes?

Reply:
>Another idea is backup the manifests and then also stop them locally in assets/$STOPPED_DIR

Sounds good to me, implemented this in 8c4e589